src/nxdoc/nuxeo-server/rest-api/web-adapters-for-the-rest-api.md

```diff
@@ -258,35 +258,46 @@ GET /nuxeo/site/api/v1/path/{pathOfTheDoc}/@children?currentPageIndex=0&pagesize
 
 {{/panel}}{{! /multiexcerpt}}{{#> panel type='code' heading='Response'}}
 
-```
+```javascript
```

Comment:
Could we use `json` instead of `javascript`? GitHub supports `JSON` as a syntax highlight according to [linguist/languages.yml#L1904](https://github.com/github/linguist/blob/27bb41aa4d97b6f809bfe16db95258361154e547/lib/linguist/languages.yml#L1904), but I'm not Nuxeo Doc has the same mechanism.